ngraph/test/frontend/shared/src/cut_specific_model.cpp

```diff
@@ -216,3 +216,21 @@ TEST_P(FrontEndCutModelTest, testExtractSubgraph) {
                                  }) != ops.end()) << "Name not found:" << name;
     }
 }
+
+TEST_P(FrontEndCutModelTest, testSetTensorValue) {
+    ASSERT_NO_THROW(doLoadFromFile());
+    Place::Ptr place;
+    ASSERT_NO_THROW(place = m_inputModel->getPlaceByTensorName(m_param.m_tensorValueName));
+    ASSERT_NO_THROW(m_inputModel->setTensorValue(place, &m_param.m_tensorValue[0]));
+
+    std::shared_ptr<ngraph::Function> function;
+    ASSERT_NO_THROW(function = m_frontEnd->convert(m_inputModel));
+    auto ops = function->get_ordered_ops();
+
+    // Ensure that it doesn't contain old input
+    auto name = m_param.m_op_before_name;
+    EXPECT_TRUE(std::find_if(ops.begin(), ops.end(),
```

Comment:
It's not clear how 'setTensorValue' correlates with expectation of not having old input...
Can we check instead that tensor value of converted ngraph::Function matches with m_param.m_tensorValue[0]?

Reply:
Done